jwst/jump/tests/test_jump_step.py

```diff
@@ -245,3 +245,35 @@ def test_two_group_integration(generate_miri_reffiles, max_cores, setup_inputs):
     out_model = JumpStep.call(model1, override_gain=override_gain,
                               override_readnoise=override_readnoise, maximum_cores=max_cores)
     assert(out_model.meta.cal_step.jump == 'SKIPPED')
+
+@pytest.mark.parametrize("max_cores", MAXIMUM_CORES)
```

Comment:
Because this test is simply dealing with the mechanics of skipping when NGROUPS is below the required threshold does it really need to execute using all available settings of MAXIMUM_CORES? Wouldn't just 1 be sufficient? The number of cores should have no effect on the skipping logic.

Reply:
I agree. I was just following the pattern of the other test cases. I'll change it.